site/src/img/laptop_illo.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="561px" height="390px" viewBox="0 0 561 390" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>Laptop Illo</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Mannequin.io-" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
@romario-agc - Not sure if you have a way to get rid of these IDs in the SVGs, but they can be a little problematic when we use them.  Let's chat about this tomorrow!